services/fluxmonitor.go

```diff
@@ -141,8 +141,6 @@ func NewFluxMonitor(config FluxMonitorConfig, triggerJobRun chan subscriber.Even
 	}
 
 	fm.canSubmitUpdated()
-	// make an initial submission on startup
-	logger.ErrorIf(fm.checkAndSendJob(false))
```

Comment:
Removed as we probably don't want node ops to waste money on gas each time they restart their EI anyways